### DISCUSSION TITLE
Statistics Page UI/UX enhancements

### DISCUSSION THREAD

**Discussion by bronwyncombs** [specify/specify7#6006] [Feature Requests]
From [Discourse: ](https://discourse.specifysoftware.org/t/changing-taxon-tile-and-statistics-panel-behaviors/2167)

**Is your feature request related to a problem? Please describe.**

> Allowing the [Statistics page](https://discourse.specifysoftware.org/t/statistics-in-specify-7/1135) to be the homepage – I thought that I read somewhere that this was possible, but can’t find any documentation of it. If it’s not possible, it should be!
> 
> Allowing users to re-order stats panels (either by editing a number field, or dragging and dropping). The fact that one needs to remake all the panels in order to switch the order they display in is really cumbersome.
> 
> Allowing users to more directly interface with calculated queries in the stats panels. Currently, the stock % calculation queries cannot be altered these queries are too one-size-fits all to be very useful – my collection displays “84+% imaged” because most lots have an attachment record that links to a ledger scan. But I can’t make a % based query that shows the proportion of lots that have specific types of attachment associated with them (i.e. a photograph of the specimen, a photograph of the labels, an SEM, an x-ray, etc).

**Describe the solution you'd like**

- allow for adding statistics to homepage; (matching or enhancing sp6 capability, see #997)
- allow to reorder stats categories with either drag-and-drop or entering order value per category
- more granular calculated query data

**Reported By**
Nate @ OSU Mollusks

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/changing-taxon-tile-and-statistics-panel-behaviors/2167/2

**Comment by grantfitzsimmons** [comment]
Split adding statistics to homepage into an issue: https://github.com/specify/specify7/issues/7529

**Comment by bhumikaguptaa** [comment]
> When a statistical ‘profile’ is created, whether it is shared or private, it would be very useful if it was possible to download this profile and share it either in another collection or with colleagues.
For example, I have created precise statistics on my profile to meet the needs of a particular project.
I would like to share this personal statistics profile named “DTE” only with the two or three people involved in the project. This would avoid my colleagues having to recreate this statistics profile on their own profiles.

Additional Enhancement requested by:
Marion Depraetere, Museum für Naturkunde Berlin, https://discourse.specifysoftware.org/t/statistics-page/1135/5
